Makefile

```diff
@@ -1,15 +1,5 @@
 SHELL = /bin/bash
 
-PATH:=$(PATH):$(GOPATH)/bin
-
-include $(shell curl --silent -o .build-harness "https://raw.githubusercontent.com/cloudposse/build-harness/master/templates/Makefile.build-harness"; echo .build-harness)
```

Comment:
need to add:
```
-include $(shell curl -sSL -o .build-harness "https://git.io/build-harness"; echo .build-harness)
```

since all the travis commands in `.travis.yml` use the Make targets from `build-harness`

Reply:
done

Reply:
in `.travis.yml`, the Make targets are for `godep` and Glide.
They need to be updated to use Go moudles.

This

```
install:
- make init
- make travis/docker-login
- make go/deps-build
- make go/deps-dev
- make go-get

script:
- make go/deps
- make go/test
- make go/lint
- make go/build-all
- ls -l release/
- make docker/build

```

needs to be updated to this:

```
install:
- make init
- make travis/docker-login
- make go/deps-dev

script:
- make go/lint
- make go/build-all
- ls -l release/
- make docker/build
```

Reply:
done;
Looks like the `build-harness` dir has `modules/go/Makefile.style` which defines a rule for `go/lint` which uses `go vet` in a no longer supported way. To fix this, I would need to probably add a new rule go-mod/lint which would invoke rule go-mod/vet. Before I go do that, is there another suggestion? @aknysh

Reply:
you can add the Make target to this repo's Makefile, and call it from `.travis.yml` (and remove `make go/lint`)

Reply:
@aknysh good point -- done.